content/source/docs/cloud/run/cli.html.md

```diff
@@ -111,78 +98,31 @@ If you use a combined repository and [specify a working directory on workspaces]
 
 CLI-driven runs upload an archive of your configuration directory
 to Terraform Cloud. If the directory contains files you want to exclude from upload,
-you can do so by defining a [`.terraformignore` file in your configuration directory][remote].
+you can do so by defining a [`.terraformignore` file in your configuration directory](/docs/cli/using-terraform-cloud/initialization.html).
 
 ## Remote Speculative Plans
 
-To run a [speculative plan][] on your configuration, use the `terraform plan` command. The plan will run in Terraform Cloud, and the logs will stream back to the command line along with a URL to view the plan in the Terraform Cloud UI.
+You can run speculative plans in any workspace where you have permission to queue plans. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
 
-Users can run speculative plans in any workspace where they have permission to queue plans. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
+To run a [speculative plan][] on your configuration, use the `terraform plan` command. The plan will run in Terraform Cloud, and the logs will stream back to the command line along with a URL to view the plan in the Terraform Cloud UI.
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
 Speculative plans use the configuration code from the local working directory, but will use variable values from the specified workspace.
 
-```
-$ terraform plan
-
-Running plan in the remote backend. Output will stream here. Pressing Ctrl-C
-will stop streaming the logs, but will not stop the plan running remotely.
-To view this plan in a browser, visit:
-https://app.terraform.io/app/my-org/my-app-dev/runs/run-LU3uk79BE5Uj77io
-
-Waiting for the plan to start...
-
-Terraform v0.11.9
-
-Configuring remote state backend...
-Initializing Terraform configuration...
-Refreshing Terraform state in-memory prior to plan...
-The refreshed state will be used to calculate this plan, but will not be
-persisted to local or remote state storage.
-
-[...]
-
-Plan: 1 to add, 0 to change, 0 to destroy.
-```
 
 ## Remote Applies
 
-When configuration changes are ready to be applied, use the `terraform apply` command. The apply will start in Terraform Cloud, and the command line will prompt for approval before applying the changes.
+You can trigger remote applies in any workspaces where you have permission to apply runs. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
```

Comment:
Not quite true! 

```suggestion
In workspaces that aren't connected to a VCS repository, users with permission to apply runs can trigger remote applies with the CLI. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
```

The reasons for this are too complicated to get into right now, but that's how it currently is.